tests/Microsoft.Bot.Builder.Tests/Teams/TeamsInfoTests.cs

```diff
@@ -301,7 +302,7 @@ private async Task CallSendMessageToTeamsChannelAsync(ITurnContext turnContext)
             {
                 var message = MessageFactory.Text("hi");
                 var channelId = "channelId123";
-                var creds = new MicrosoftAppCredentials("big-guid-here", "appPasswordHere");
+                var creds = new MicrosoftAppCredentials(string.Empty, string.Empty);
```

Comment:
```suggestion
                var creds = MicrosoftAppCredentials.Empty;
```

Reply:
surprisingly this doesn't work everywhere - the create conversation call needs new MicrosoftAppCredentials(string.Empty, string.Empty) otherwise you get a null exception. We can fix that one in Cloud Adapter